### DISCUSSION TITLE
Running multiple servers for a major mode

### DISCUSSION THREAD

**Discussion by fast-90** [joaotavora/eglot#1221] [Q&A]
Hi all,

Is it possible to run multiple servers for a major mode with eglot? 

Case in point: for Python I am currently using Pyright with eglot, however I want to enable [ruff-lsp](https://github.com/charliermarsh/ruff-lsp) which catches some of the errors not caught by Pyright.

https://github.com/joaotavora/eglot/discussions/1122#discussioncomment-4268244 suggests that it was not possible back then. Is this still the case?

Other discussions or issues I found without a definitive answer:
- https://github.com/joaotavora/eglot/issues/90 (I believe `eglot-alternatives` is meant to look for the multiple language server, and picks only the first one it can find in the list of alternatives).
- https://github.com/joaotavora/eglot/issues/423
- https://github.com/joaotavora/eglot/discussions/681

**Comment by Gavinok** [comment]
@fast-90 I do not believe so no. As of now it's a commonly requested feature but there doesn't seem to be any active work being done to add it.
